haystack/document_stores/memory.py

```diff
@@ -480,6 +480,19 @@ def get_document_count(
         )
         return len(documents)
 
+    def update_document_meta(self, id: str, meta: Dict[str, Any], index: str = None):
```

Comment:
Can we make that now an abstractmethod of `BaseDocumentStore`? I think the only document store that does not support it would be `DeepsetCloudDocumentStore` and given its read-only nature that is not a show-stopper here.

Reply:
Added it to `BaseDocumentStore`